compute/cker/src/Mul.test.cc

```diff
@@ -187,6 +187,70 @@ TEST(CKer_Operation, Mul)
       EXPECT_EQ(output[i], expected_output[i]);
   }
 
+  // Int64
+  {
+    // Shape: {1, 2, 2, 1}
+    std::vector<int64_t> input1 = {10, 9, -11, 3};
+    // Shape: {1, 2, 2, 1}
+    std::vector<int64_t> input2 = {2, -2, -3, 4};
+    std::vector<int64_t> expected_output = {20, -18, 33, 12};
```

Comment:
(optional) It would be better to use bigger number which can be calculated with int64.
```suggestion
    std::vector<int64_t> input1 = {INT32_MIN, 9, -11, 3};
    // Shape: {1, 2, 2, 1}
    std::vector<int64_t> input2 = {INT32_MIN, -2, -3, 4};
    std::vector<int64_t> expected_output = {INT64_MIN, -18, 33, 12};
```

Reply:
I'll update test later